tasks/libs/requirements-stats.txt

```diff
@@ -1,2 +0,0 @@
-invoke==2.2.0
-datadog-api-client==2.3.0
```

Comment:
This doesn't appear to be installed in the `datadog-ci-uploader` image, but it's used when computing the stats

Reply:
you are right I missed some dependencies, I am taking care of them with [this](https://github.com/DataDog/datadog-agent-buildimages/pull/438)